recipes/mender-monitor/debian-master/rules

```diff
@@ -5,5 +5,8 @@
 %:
 	dh $@
 
-override_dh_link:
-	true
+override_dh_auto_install:
+	DESTDIR=debian/tmp make install install-example-monitors
+
+override_dh_missing:
+	dh_missing --fail-missing
```

Comment:
:thinking: This will catch missing files as I worried about above?

I didn't know about this but sounds like a nice idea! The only drawback is that we will often need to update the recipes and create more `debian-X.X.X` copies.

Reply:
Yes, I would vote for giving it a try though, I actually was missing a file, after the original install.

What you think?

Reply:
Yes, let's do it this way. I like it too.